README.md

```diff
@@ -154,32 +149,28 @@ We recommend either Ruby 2.3 or the latest 2.2 version.
 
 # Creating a Sufia-based app
 
-## Rails
+## Redis
 
-Generate a new Rails application. We recommend the latest Rails 5.0 or 4.2 release.
+[Redis](http://redis.io/) is a key-value store that Sufia uses to provide activity streams on repository objects and users, and to prevent race conditions as a global mutex when modifying order-persisting objects.
 
-```
-gem install rails -v 5.0.0.1
-rails new my_app
-```
+Starting up Redis will depend on your operating system, and may in fact already be started on your system. You may want to consult the [Redis documentation](http://redis.io/documentation) for help doing this.
 
-## Sufia's Ruby-related dependencies
+## Rails
 
-Add the following lines to your application's Gemfile.
+Generate a new Rails application. We recommend the latest Rails 5.0 or 4.2 release.
 
 ```
-gem 'sufia', '7.2.0'
+# If you don't already have Rails at your disposal...
+gem install rails -v 5.0.0.1
+rails new my_app -m https://raw.githubusercontent.com/projecthydra/sufia/master/template.rb
```

Comment:
@barmintor @cgalarza I dig the idea, but I can't think of a good way to handle this. If I point at, say, the 7.2.0 release, the README will be broken because `template.rb` does not exist in the 7.2.0 release. I'm not sure I want to cut a special release just to fix up the README, though I can be convinced otherwise. Pointing at a release branch could work, but we tend to create those at point of need (when someone wants something backported). :thinking:

What if we leave it pointing at `master` now, and use our release management process to edit this line when we cut our next release? I have already edited the release mgmt notes to reflect that cutting a new release should at least require reviewing, if not modifying, the README and `template.rb`.